devtools/messenger/BUILD

```diff
@@ -0,0 +1,21 @@
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@rules_player//javascript:defs.bzl", "js_pipeline")
+load("//helpers:defs.bzl", "tsup_config", "vitest_config")
+
+npm_link_all_packages(name = "node_modules")
+
+tsup_config(name = "tsup_config")
+
+vitest_config(name = "vitest_config")
+
+js_pipeline(
+    package_name = "@player-tools/devtools-messenger",
+    peer_deps = [
+        "//:node_modules/typescript",
```

Comment:
Why does this have a dep on TS?

Reply:
That was one of the first BUILD files, and I probably got it there following some old pattern. It's not necessary; I removed it.